src/08_comprehensions.py

```diff
@@ -36,6 +36,6 @@
 x = input("Enter comma-separated numbers: ").split(',')
 
 # What do you need between the square brackets to make it work?
-y = []
+y = [int(n) for n in x if not int(n) & 1]
```

Comment:
oh that's fun. I feel like I only see (and do) `%2 == 0`

Reply:
Yeah bitwise is fun, though I think I'll have to be cautious if I ever encounter a new data type. It seems to work well enough with integers and decimals in my experience at least.